config/management/src/lib.rs

```diff
@@ -105,6 +116,16 @@ impl Command {
         }
     }
 
+    pub fn set_layout(self) -> Result<crate::layout::Layout, Error> {
+        if let Command::SetLayout(set_layout) = self {
+            set_layout.execute()
+        } else {
+            let expected = CommandName::SetLayout.to_string();
```

Comment:
nit: can we maybe refactor this into a helper method, we're using the same code 5 times (saving lines 😄). Also, can we update the verify() method below to use the same pattern?

Reply:
Welcome to the world of Rust

I was planning on refactoring verify to support verify-local and verify-remote. That way we don't need to manually inspect if genesis is ready to be built.

Reply:
Are you going to refactor verify in a future PR? Also, can we not get rid of this pattern with a simple helper? e.g.,

fn unexpected_command(self, expected: CommandName) -> Error {
  let actual = CommandName::from(&self).to_string();
  Err(Error::UnexpectedCommand(expected.to_string(), actual))
} 

then each caller:

self.unexpected_command(<EXPECTED_COMMAND>)